app/javascript/app/components/button/button-component.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import Link from 'redux-first-router-link';
+import { themr } from 'react-css-themr';
+
+import styles from './button-styles.scss';
+
+const Button = props => {
+  const { link, href, children, className, disabled, onClick, theme } = props;
+  const classNames = cx(className, theme.button, {
+    [theme.disabled]: !onClick && !link && !href
```

Comment:
Maybe the `disabled` feature of the `<Button>` component is not yet needed ? What do you think?

Reply:
Its true but I think we will need it in some cases as after saving ... And its a basic status of a button so I wont say is bad to have it by default